bazel/third_party/aes_siv/aes_siv.BUILD

```diff
@@ -17,4 +17,5 @@ cc_library(
     includes = ["."],
     visibility = ["//visibility:public"],
     deps = ["@openssl"],
+    copts = ["-O3", "-g"],
```

Comment:
The combination of `-O3` and `-g` causes problems on macOS--see https://github.com/urbit/vere/issues/131#issuecomment-1398626261 --so this should instead be:
```python
copts = ["-O3"] + select({
    "@platforms//os:macos": [],
    "//conditions:default": ["-g"],
})
```

Reply:
done